frontend/src/components/EntityList.vue

```diff
@@ -1,104 +1,56 @@
-    </div>
-    <div class="col-lg-10 table-responsive">
-      <h2 v-if="this.selectedSchema" class="mt-1">
-        Schema {{ this.selectedSchema.name }}
-        <RouterLink
-          v-if="this.selectedSchema"
-          :to="`/${this.selectedSchema.slug}`"
-          class="btn btn-sm btn-primary"
-          style="text-decoration: none"
-          >View schema</RouterLink
-        >
-        <RouterLink
-          v-if="this.selectedSchema"
-          :to="`/edit/${this.selectedSchema.slug}`"
-          class="btn btn-sm btn-primary ms-1"
-          style="text-decoration: none"
-          >Edit schema</RouterLink
-        >
-        <RouterLink
-          :to="`/${this.selectedSchema?.slug}/entities/new`"
-          class="btn btn-sm btn-primary ms-1"
-          style="text-decoration: none"
-          >Create new entity</RouterLink
-        >
-      </h2>
-      <SearchPanel
-        @search="setFiltersAndSearch"
-        :key="this.selectedSchema?.slug"
-        :filterableFields="this.filterableFields"
-        :operators="this.operators"
-      />
-      <Pagination
-        v-if="this.totalEntities > this.entitiesPerPage"
-        v-on:goTo="changePage"
-        :totalEntities="this.totalEntities"
-        :entitiesPerPage="this.entitiesPerPage"
-        :currentPage="this.currentPage"
-      />
-
-      <div class="row">
-        <div class="col-11">
-          <label for="entitiesLimit"><small>Entities per page</small></label>
-          <select v-model="entitiesPerPage" id="entitiesLimit">
-            <option>10</option>
-            <option>30</option>
-            <option>50</option>
-          </select>
-        </div>
-        <div class="col">
-          <p>
-            <small>{{ this.totalEntities }} results</small>
-          </p>
-        </div>
-      </div>
+  <SearchPanel
+      @search="setFiltersAndSearch"
+      :key="schema?.slug"
+      :filterableFields="filterableFields"
+      :operators="operators"/>
+  <Pagination
+      v-if="totalEntities > entitiesPerPage"
+      v-on:goTo="changePage"
+      :totalEntities="totalEntities"
+      :entitiesPerPage="entitiesPerPage"
+      :currentPage="currentPage"/>
 
-      <EntityListTable
-        @reorder="this.reorder"
-        :entities="this.entities"
-        :schemaSlug="this.selectedSchema?.slug"
-      />
-      <Pagination
-        v-if="this.totalEntities > this.entitiesPerPage"
-        v-on:goTo="changePage"
-        :totalEntities="this.totalEntities"
-        :entitiesPerPage="this.entitiesPerPage"
-        :currentPage="this.currentPage"
-      />
+  <div class="row mb-1">
+    <div class="col-2">
+      <label for="entitiesLimit"><small>Entities per page</small></label>
+    </div>
+    <div class="col-1">
+      <select v-model="entitiesPerPage" id="entitiesLimit" class="form-select form-select-sm">
+        <option>10</option>
+        <option>30</option>
+        <option>50</option>
+      </select>
+    </div>
+    <div class="col-9 text-end">
```

Comment:
I'd add something like `col-sm-2` for first div and `col-sm-8` for second one because at ~982px of width you can't really see what's selected

Reply:
Right. I'd go even further and say that `col`s don't fit well here. I've replaced it with a `d-flex` :grin: